core/ir/instr.h

```diff
@@ -2589,6 +2589,16 @@ instr_t *
 instr_create_0dst_4src(dcontext_t *dcontext, int opcode, opnd_t src1, opnd_t src2,
                        opnd_t src3, opnd_t src4);
 
+DR_API
+/**
+ * Convenience routine that returns an initialized instr_t allocated
```

Comment:
I know the others are missing it too: but including a `#` (so `#instr_t`) will add a Doxygen link.